scripts/multisrc/madara/sources.json

```diff
@@ -368,5 +368,85 @@
       "useNewChapterEndpoint": true,
       "lang": "Thai"
     }
+  },
+  {
+    "id": "azora",
+    "sourceSite": "https://azorago.com/",
+    "sourceName": "Azora",
+    "options": {
+      "useNewChapterEndpoint": true,
+      "lang": "Arabic"
+    }
+  },
+  {
+    "id": "translatinotaku",
+    "sourceSite": "https://translatinotaku.net/",
+    "sourceName": "Translation Otaku",
+    "options": {
+      "useNewChapterEndpoint": true,
+      "lang": "English"
+    }
+  },
+  {
+    "id": "dragontea",
+    "sourceSite": "https://dragontea.ink/",
+    "sourceName": "Dragon Tea",
+    "options": {
+      "useNewChapterEndpoint": true,
+      "lang": "English"
+    }
+  },
```

Comment:
Dragon tea doesn't work without changing some stuff in the generator, I was working on it

Reply:
lmao, they are mangas

Reply:
I think we should have path options for some sites like this. Novels still work fine

![Screenshot_20240429-155446](https://github.com/LNReader/lnreader-plugins/assets/86464880/ca985fc7-6d97-4a70-9f7f-e829e2623cae)

Reply:
oh, it doesnt, what is this language XD